pyproject.toml

```diff
@@ -33,9 +33,9 @@ docs = [
 ]
 
 [project.urls]
-homepage = "https://github.com/ssec-jhu/base-template"
-documentation = "https://github.com/ssec-jhu/base-template"
-repository = "https://github.com/ssec-jhu/base-template"
+homepage = "git@github.com:ssec-jhu/flfm-ij-plugin.git"
+documentation = "git@github.com:ssec-jhu/flfm-ij-plugin.git"
```

Comment:
@ryanhausen These aren't correct.

Reply:
I think its because I use ssh to clone. I added an issue:

https://github.com/ssec-jhu/base-template/issues/184